src/components/BackToTopButton/index.styles.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from '../../helpers/style';
+
+export const BackToTopStyles = styled.div`
+  display: block;
+  z-index: 99;
+  position: fixed;
+  bottom: -50px;
+  right: 5px;
+  background: ${props => props.theme.colors['blue']};
+  padding: 20px 5px 10px 5px;
+  border-radius: 5px;
+  box-shadow: 0px 0px 1px 0px rgba(0, 0, 0, 0.6);
+  cursor: pointer;
+  transition: all ease-in-out 0.5s;
+
+  &.open {
+    bottom: 5px;
```

Comment:
Will this interfere with the standard footer? We might want to think about that, but this looks good as it is currently

Reply:
Not to my understanding, you can npm start and try it. But it does have a same background color as the most right footer link, hence it needs somekind of shadow or border.  
Hmm. or do you mean is it on top of the footer link, yes. Maybe it does need a little tweaking...?

Reply:
@meiamsome I would say it's not in the way of the shiffman.net link, plenty of room to click that.  

![2019-01-26_0439](https://user-images.githubusercontent.com/22402899/51781673-af97a100-2124-11e9-88b6-b41905a367d7.png)
![2019-01-26_0440](https://user-images.githubusercontent.com/22402899/51781675-b32b2800-2124-11e9-824e-eb2659867a7d.png)

Reply:
Addendum: just found [scrollintoview](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView), so we don't need the sweet scroll-library at all if we want to have a 'scroll back to top' instead of just 'jump back to top'.  Which way do we go and does the button itself need any more tampering?